traveler-ios-ui/TravelerKitUI/Models/ItineraryByDay.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  ItineraryByDay.swift
+//  TravelerKitUI
+//
+//  Created by Rakin Hoque on 2019-12-04.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import Foundation
+import TravelerKit
+
+// TODO: Think of a way to make this more generic and if possible move into Core
+public class ItineraryByDay {
```

Comment:
structs over classes.

Reply:
None of our view models are structs, why does this one need to be?  Will require many changes to change this to a struct as is.